moped-editor/src/utils/gridTableHelpers.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Seeing how we are using this function now in 3 places, I'm moving it out to its own file.